config/locales/en.yml

```diff
@@ -343,6 +343,10 @@ en:
           attributes:
             appropriate_body_id:
               inclusion: Choose an organisation
+        delivery_partners/choose_organisation_form:
+          attributes:
+            delivery_partner_id:
+              inclusion: Choose an organisation
```

Comment:
should this be choose a delivery partner 🤔 ?

Reply:
@cwrw 
In the UI design we ask them to choose an organisation. The error message should stay consistent